blip-tap.md

```diff
@@ -939,7 +940,10 @@ where:
 * `rfq_id` is a randomly generate 32-byte value to uniquely identify this RFQ
   request
 * `asset_id` is the asset ID of the asset the receiver intends to receive
-* `asset_amt` is the amount of units of said asset
+* `swap_type` is the type of swap the message sender is requesting. When the
+  `swap_type` is `1`, the receiver is requesting a swap of their Tap asset for
```

Comment:
`the receiver is requesting a swap`
shouldn't this be `the sender is requesting a swap`?

Also in the line below.

Reply:
Yes, you're correct. Thank you! Updated.